buzzad-benefit/app/src/main/java/com/buzzvil/buzzad/benefit/sample/publisher/MainActivity.java

```diff
@@ -85,6 +87,12 @@ public void onClick(View v) {
             NativeCarouselActivity.startActivity(MainActivity.this);
         });
 
+        this.nativeAd2Button = findViewById(R.id.native_ad_2_button);
+        nativeAd2Button.setOnClickListener(v -> {
+            Intent intent = new Intent(MainActivity.this, NativeAd2Activity.class);
+            startActivity(intent);
+        });
```

Comment:
같은 파일에 다른 clickListener 와 동일하게 되어있으면 좋을 것 같습니다.

Reply:
캐러셀을 제외한 다른 버튼들은 프래그먼트를 사용하고 있습니다.
액티비티 전환을 하는 경우는 캐러셀 버튼밖에 없는데, 캐러셀은 아래와 같이 스태틱 메서드로 startActivity를 미리 구성해서 사용하고 있습니다. 이렇게 복잡하게 할 필요는 없는 것 같아서 네이티브 2.0 버튼 동작은 setOnClickListener에 간단하게 작성했습니다. 캐러셀 액티비티 처럼 구성하는게 낫겠다고 생각하시면 변경하겠습니당

```
// NativeCarouselActivity.kt
class NativeCarouselActivity : AppCompatActivity() {

    companion object {
        private const val TAG = "NativeCarouselActivity"

        @JvmStatic
        fun startActivity(context: Context) {
            val intent = Intent(context, NativeCarouselActivity::class.java)
            context.startActivity(intent)
        }
    }

// MainActivity.java
nativeCarouselButton.setOnClickListener(v -> {
    NativeCarouselActivity.startActivity(MainActivity.this);
});
```

Reply:
nit 에 가까운 제안이었는데, 제가 설명을 너무 못적었군요.. 죄송함다 :) 
설명해주신 부분은 그대로 두어도 괜찮을것 같아요 ! 
자세한 설명 감사합니당
```suggestion
        nativeAd2Button.setOnClickListener(new View.OnClickListener() {
            Intent intent = new Intent(MainActivity.this, NativeAd2Activity.class);
            startActivity(intent);
        });
```